addons/web/static/src/js/views/kanban/kanban_model.js

```diff
@@ -472,56 +472,39 @@ var KanbanModel = BasicModel.extend({
         }
         return Promise.resolve();
     },
-    _reloadProgressBarFromRecord: function (recordID, def) {
-        const self = this;
-        let element = this.localData[recordID];
-        while (element) {
-            if (element.progressBar) {
-                return def.then(data => {
-                    return self._readProgressBar(element).then(() => data);
-                });
-            }
-            element = this.localData[element.parentID];
-        }
-        return def;
-    },
     /**
      * Reloads all progressbar data. This is done after given promise and
-     * insures that the given promise's result is not lost.
+     * ensures that the given promise's result is not lost.
```

Comment:
We do not sell insurances to the caller functions :+1:

Reply:
haha